pgsql/pc_pgsql.c

```diff
@@ -799,16 +806,43 @@ pc_patch_serialize_to_uncompressed(const PCPATCH *patch_in)
 	return serpatch;
 }
 
+static void
+pc_patch_set_common(PCPATCH *patch, const SERIALIZED_PATCH *serpatch, const PCSCHEMA *schema)
```

Comment:
I propose to rename this as `pc_patch_common_deserialize`.
It should return somehow the data pointer to the payload, since we cannot rely on serpatch->data for patches in version 0.